x-pack/auditbeat/Jenkinsfile.yml

```diff
@@ -2,6 +2,7 @@ when:
     branches: true             ## for all the branches
     changeset:                 ## when PR contains any of those entries in the changeset
         - "^x-pack/auditbeat/.*"
+        - "^auditbeat/.*"      ## when changes in the auditbeat
```

Comment:
I guess this cannot be added for linting only, so we don't have to add the negation in all the other builds?

Reply:
I could not find a different approach with the existing declarative syntax :(

Although,I just found something that I wanna give a go in another PR

Reply:
My new proposal -> https://github.com/elastic/beats/pull/23695